frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-patient-count/radiogram-card-content-patient-count.component.ts

```diff
@@ -15,39 +20,44 @@ import {
     styleUrls: ['./radiogram-card-content-patient-count.component.scss'],
 })
 export class RadiogramCardContentPatientCountComponent implements OnInit {
-    @Input() radiogram!: PatientCountRadiogram;
+    @Input() radiogramId!: UUID;
+    radiogram$!: Observable<PatientCountRadiogram>;
     bluePatientsEnabled$!: Observable<boolean>;
-    simulatedRegionName$!: Observable<string>;
-    totalPatientCount!: number;
+    totalPatientCount$!: Observable<number>;
 
     constructor(private readonly store: Store<AppState>) {}
 
     ngOnInit(): void {
-        this.totalPatientCount =
-            this.radiogram.patientCount.black +
-            this.radiogram.patientCount.white +
-            this.radiogram.patientCount.red +
-            this.radiogram.patientCount.yellow +
-            this.radiogram.patientCount.green +
-            this.radiogram.patientCount.blue;
+        const radiogramSelector = createSelectRadiogram(
+            this.radiogramId
+        ) as MemoizedSelector<
+            AppState,
+            NonNullable<PatientCountRadiogram>,
+            (s1: {
+                [key: string]: ExerciseRadiogram;
+            }) => NonNullable<PatientCountRadiogram>
+        >;
```

Comment:
Please: Move this to the `store.select` call, then you would also not need such a complicated cast that is decreasing readability even more. (You can cast the result of `store.select` if necessary)

Reply:
Ah, then I must have missed that one when creating the radiograms. Do you mind moving it to `frontend/src/app/state/application/selectors/exercise.selectors.ts`? We also have a special `selectPropertyFactory` function there, that is correctly typed afaik

Reply:
I don't understand what you mean by this... I use the radiogram selector twice because I want to select the radiogram (first usage) and to cap
Duplicate of encapsulate a complicated computation that I think should not happen in the html file (second usage) This means moving this inside a store.select would duplicate code

Reply:
Yeah nevermind, missread, my bad lol